docker-hub/official_images.md

```diff
@@ -3,49 +3,33 @@ description: Guidelines for Official Images on Docker Hub
 keywords: Docker, docker, registry, accounts, plans, Dockerfile, Docker Hub, docs, official,image, documentation
 title: Docker Official Images
 redirect_from:
-- /docker-hub/official_repos/
+  - /docker-hub/official_repos/
 ---
 
 The [Docker Official Images](https://hub.docker.com/search?q=&type=image&image_filter=official){:target="_blank" rel="noopener" class="_"} are a
-curated set of Docker repositories hosted on Docker Hub. They are
+curated set of Docker repositories hosted on Docker Hub. They're
 designed to:
 
-* Provide essential base OS repositories (for example,
-  [ubuntu](https://hub.docker.com/_/ubuntu/){:target="_blank" rel="noopener" class="_"},
-  [centos](https://hub.docker.com/_/centos/){:target="_blank" rel="noopener" class="_"}) that serve as the
+- Provide essential base OS repositories (for example,
+  [Ubuntu](https://hub.docker.com/_/ubuntu/){:target="_blank" rel="noopener" class="_"},
+  [CentOS](https://hub.docker.com/_/centos/){:target="_blank" rel="noopener" class="_"}) that serve as the
   starting point for the majority of users.
-
-* Provide drop-in solutions for popular programming language runtimes, data
-  stores, and other services, similar to what a Platform as a Service (PAAS)
-  would offer.
-
-* Exemplify [`Dockerfile` best practices](/engine/userguide/eng-image/dockerfile_best-practices/)
+- Provide drop-in solutions for popular programming language runtimes, data
+  stores, and other services.
+- Exemplify [`Dockerfile` best practices](/engine/userguide/eng-image/dockerfile_best-practices/)
   and provide clear documentation to serve as a reference for other `Dockerfile`
   authors.
-
-* Ensure that security updates are applied in a timely manner. This is
+- Ensure that security updates are applied in a timely manner. This is
   particularly important as Docker Official Images are some of the most
   popular on Docker Hub.
 
-Docker, Inc. sponsors a dedicated team that is responsible for reviewing and
-publishing all content in the Docker Official Images. This team works in
-collaboration with upstream software maintainers, security experts, and the
-broader Docker community.
+Images part of this program have a special badge on Docker Hub making it easier for you to identify projects that are official Docker images.
```

Comment:
"Images part of this program" feels like it's missing something - "Images _that are_ part", perhaps?